e2e/terraform/gke/variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "credentials" {
+  default = "account.json"
+}
+
+variable "region" {
+  default = "us-central1-a"
+}
+
+variable "project" {
+  default = "strong-eon-153112"
```

Comment:
It may be a good idea to use place-holder names here? or ENV?

Reply:
yes @kmova , will use ENV